.github/workflows/auto-merge.yaml

```
@@ -0,0 +1,30 @@
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+  check_suite:
+    types:
+      - complete
```

actionlint (1.7.7)

invalid activity type "complete" for "check_suite" Webhook event. available types are "completed"

(events)